samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -113,12 +115,7 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
     return <Spinner label={'Creating adapter'} ariaLive="assertive" labelPosition="top" />;
   }
 
-  let callInvitationUrl: string | undefined = window.location.href;
-  /* @conditional-compile-remove(rooms) */
-  // If role is defined then the call is a Rooms call so we should not make call invitation link available
-  if (role) {
-    callInvitationUrl = undefined;
-  }
+  const callInvitationUrl: string | undefined = window.location.href;
```

Comment:
This seems to be changing business logic of when to show `callInvitationUrl`.

Reply:
My bad. Reverted.